internal/adapters/entrypoints/rest/common.go

```diff
@@ -135,17 +135,27 @@ func DecodeRequest[T any](w http.ResponseWriter, req *http.Request, body *T) err
 }
 
 func getValidationMessage(field validator.FieldError) string {
+	if field.Field() == "FeePercentage" {
```

Comment:
I think is better to have this kind of handling of the error msg in the front. In there you have the context of which field is being manipulated and can provide the proper msg on each case. If you go this way this function will become huge the more specific you want to be in the error msg. If any, maybe you can return something similar to the previous version to indicate the field and the rule being broken, but this level of responsibility is too much for this function